modules/luci-mod-system/luasrc/model/cbi/admin_system/system.lua

```diff
@@ -105,6 +106,29 @@ o:value(8, translate("Normal"))
 o:value(9, translate("Warning"))
 
 
+--
+-- Zram Properties
+--
+if has_zram then
+	o = s:taboption("zram", Value, "zram_size_mb", translate("ZRam Size"), "Mb")
+	o.optional    = true
+	o.placeholder = 16
+	o.datatype    = "uinteger"
+	
+	o = s:taboption("zram", ListValue, "zram_comp_algo", translate("ZRam Compression Algorithm"))
+	o.optional    = true
+	o.placeholder = lzo
+	o:value("lzo", "lzo")
+	o:value("lz4", "lz4")
+	o:value("deflate", "deflate")
+	
+	o = s:taboption("zram", Value, "zram_comp_streams", translate("ZRam Compression Streams"), translate("Number of core to be used for compression"))
```

Comment:
Alternative suggestion for the help text:
`Number of parallel threads used for compression`